demo/src/utils/editor-history.util.ts

```diff
@@ -0,0 +1,17 @@
+const AVRJS8_EDITOR_HISTORY = 'AVRJS8_EDITOR_HISTORY';
+
+export class EditorHistoryUtil {
+  static storeSnippet(codeSnippet: string) {
+    if (window.localStorage) {
+      window.localStorage.setItem(AVRJS8_EDITOR_HISTORY, codeSnippet);
+    } else throw new Error('no localStorage support');
```

Comment:
If there's no localStorage, it would make more sense to silently ignore the error instead of throwing, so that the demo doesn't break. 

We also need to add this check to getValue/clearSnippet

Reply:
your'e right! I'll add the guard